.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
I like having this run on `branches: '**'` so that PRs targeting other PR branches (i.e. stacked changes) still get CI run.